packages/webb-ui-components/src/components/Chip/Chip.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { useMemo } from 'react';
+import { twMerge } from 'tailwind-merge';
+
+import { ChipProps } from './types';
+import { getChipClassName } from './utils';
+
+/**
+ * The `Chip` component
+ *
+ * Props:
+ *
+ * - `color`: The visual style of the badge (default: "green")
+ * - `isDisabled`: If `true`, the chip will display as disabled state
+ *
+ * @example
+ *
+ * ```jsx
+ *  <Chip>Active</Chip>
+ *  <Chip color="red" isDisabled>Disabled</Chip>
+ * ```
+ */
+export const Chip: React.FC<ChipProps> = (props) => {
+  const { children, className, color = 'green', isDisabled } = props;
+
+  const baseClsx = useMemo(() => 'box-border inline-block px-3 py-1 rounded-full font-bold body4', []);
+
+  const _className = useMemo(
```

Comment:
`_` an underscore prefix for variable means it isn't being used, can we change this to be something else or change the destructuring variable name for `className` to be `class` or smt

Reply:
@AtelyPham  other than that looks good